src/mlpack/tests/ind2sub_test.cpp

```diff
@@ -0,0 +1,19 @@
+#include <mlpack/core.hpp>
+//#include <mlpack/methods/edge_boxes/structured_tree.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "old_boost_test_definitions.hpp"
+BOOST_AUTO_TEST_SUITE(ind2sub_test);
+
+/**
+ * This tests handles the case wherein only one class exists in the input
+ * labels.  It checks whether the only class supplied was the only class
+ * predicted.
+ */
+BOOST_AUTO_TEST_CASE(ind2sub_test)
+{
+  arma::mat A = arma::randu(5,5);
+  arma::uvec u = arma::ind2sub(arma::size(A), 3);
+  u.print();
```

Comment:
Hmm, this test doesn't actually test anything, do you think that we can test the output with `BOOST_REQUIRE_EQUAL`?  I don't know what the output values of u are supposed to be, but maybe something like

```
BOOST_REQUIRE_EQUAL(u(0), 0);
BOOST_REQUIRE_EQUAL(u(1), 3);
```

Technically I don't think we need to test Armadillo support since Armadillo should already be testing that, but, one nice thing this test *does* do is ensure that ind2sub() compiles correctly.  If you like, we can also add a very simple test like this for sub2ind().